.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,25 @@
+name: Spellcheck
```

Comment:
Would it be possible to call this "spellingcheck"? Since we are checking spelling, not spells...